swingset/src/main/java/com/nqadmin/swingset/SSDBComboBox.java

```diff
@@ -1504,31 +1478,52 @@ public void setSelectedItem(final Object _value) {
 	 *
 	 * @param _value value to set as currently selected.
 	 */
+	//
+	// TODO: WHAT IS THIS SUPPOSED TO DO IF BOTH O,O2 are set?
+	//
+	// TODO: thought this was deprecated/removed. Maybe that was SSComboBox
+	// 
 	public void setSelectedStringValue(final String _value) {
 
-		// ONLY NEED TO PROCEED IF THERE IS A CHANGE
-		// TODO consider firing a property change
-		try (Model.Remodel remodel = comboInfo.getRemodel()) {
+		// TODO consider firing a property change ??? there are already events
+
+		try (Model.Remodel remodel = optionModel.getRemodel()) {
+			// TODO: If getAllowNull, might be empty, but still have 1 item.
+			// TODO: Why bother checking if empty? If it's empty,
+			//       then there is nothing expensive that happens in here.
 			if(!remodel.isEmpty()) {
```

Comment:
The code works (I think) if it's empty. Why check?

Reply:
Agree. See notes above. This used to be a one line method.